client/account-recovery/lost-password/lost-password-form/index.jsx

```diff
@@ -14,83 +14,87 @@ import Card from 'components/card';
+		this.props.fetchResetOptionsByLogin( this.props.userLogin );
 	};
 
 	onUserLoginChanged = ( event ) => {
-		this.setState( { userLogin: event.target.value } );
+		this.props.updatePasswordResetUserData( {
+			user: event.target.value
+		} );
 	};
 
-	componentWillReceiveProps( nextProps ) {
-		if ( nextProps.resetOptions ) {
-			this.setState( { isSubmitting: false } );
-		}
-	}
-
 	render() {
-		const { translate } = this.props;
-		const { isSubmitting, userLogin } = this.state;
-		const isPrimaryButtonDisabled = ! userLogin || isSubmitting;
+		const {
+			translate,
+			userLogin,
+			isRequesting,
+			requestError,
+		} = this.props;
+
+		const isPrimaryButtonDisabled = ! userLogin || isRequesting;
 
 		return (
 			<div>
-				<h2 className="lost-password-form__title">
-					{ translate( 'Lost your password' ) }
-				</h2>
-				<p>{ translate( 'Follow these simple steps to reset your account:' ) }</p>
-				<ol className="lost-password-form__instruction-list">
-					<li>
-						{ translate(
-							'Enter your {{strong}}WordPress.com{{/strong}} username or email address',
-							{ components: { strong: <strong /> } }
-						) }
-					</li>
-					<li>
-						{ translate( 'Choose a password reset method' ) }
-					</li>
-					<li>
+				<Card compact>
+					<h2 className="lost-password-form__title">
+						{ translate( 'Lost your password?' ) }
+					</h2>
+					<p>{ translate( 'Follow these simple steps to reset your account:' ) }</p>
+					<ol className="lost-password-form__instruction-list">
+						<li>
+							{ translate(
+								'Enter your {{strong}}WordPress.com{{/strong}} username or email address',
+								{ components: { strong: <strong /> } }
+							) }
+						</li>
+						<li>
+							{ translate( 'Choose a password reset method' ) }
+						</li>
+						<li>
+							{ translate(
+								'Follow instructions and be re-united with your {{strong}}WordPress.com{{/strong}} account',
+								{ components: { strong: <strong /> } }
+							) }
+						</li>
+					</ol>
+					<p>
 						{ translate(
-							'Follow instructions and be re-united with your {{strong}}WordPress.com{{/strong}} account',
-							{ components: { strong: <strong /> } }
+							'Want more help? We have a full {{link}}guide to resetting your password{{/link}}.',
+							{ components: { link: <a href={ support.ACCOUNT_RECOVERY } /> } }
 						) }
-					</li>
-				</ol>
-				<p>
-					{ translate(
-						'Want more help? We have a full {{link}}guide to resetting your password{{/link}}.',
-						{ components: { link: <a href={ support.ACCOUNT_RECOVERY } /> } }
-					) }
-				</p>
-				<Card>
+					</p>
+				</Card>
+				<Card compact>
 					<FormLabel>
 						{ translate( 'Username or Email' ) }
 
 						<FormInput
 							className="lost-password-form__user-login-input"
 							onChange={ this.onUserLoginChanged }
-							value={ userLogin }
-							disabled={ isSubmitting } />
+							value={ userLogin ? userLogin : '' }
```

Comment:
Would you even need this if we just set the default prop to "" instead of null [here](https://github.com/Automattic/wp-calypso/pull/11695/files#diff-238b2ec450850f32a848d6ce92f16ae0R117)